doc/conf.py

```diff
@@ -3,29 +3,33 @@
 # psyclone documentation build configuration file, created by
 # sphinx-quickstart on Mon Jan 27 12:50:29 2014.
 #
-# This file is execfile()d with the current directory set to its containing dir.
+# This file is execfile()d with the current directory set to its
+# containing dir.
 #
 # Note that not all possible configuration values are present in this
 # autogenerated file.
 #
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, os
+import sys
+import os
```

Comment:
Changes to make pep8 compliant. Good, but sys is not needed, as specified by pylint. There are a few pylint issues leading to a negative score but it does not really matter that much for this file.